packages/utilities/psammead-styles/package-lock.json

```
@@ -1,5 +1,5 @@
 {
   "name": "@bbc/psammead-styles",
-  "version": "2.3.0",
+  "version": "2.4.0",
```

```suggestion
  "version": "3.0.0",
```
to match `package.json`